src/mac/mod.rs

```diff
@@ -0,0 +1,21 @@
+//! Partial implementation of the IEEE 802.15.4 MAC layer
+
+mod frame;
```

Comment:
```suggestion
pub mod frame;
```
Not necessary, but doesn't hurt and it consistent with the other modules.